cirq/google/engine/engine.py

```diff
@@ -375,14 +354,19 @@ def run_sweep(
 
         # Create program.
         request = {
-            'name': 'projects/%s/programs/%s' % (job_config.project_id,
-                                                 job_config.program_id,),
-            'gcs_code_location': {'uri': job_config.gcs_program},
-            'code': code,
+            'name':
+            'projects/%s/programs/%s' % (
+                self.project_id,
+                job_config.program_id,
+            ),
+            'gcs_code_location': {
+                'uri': job_config.gcs_program
+            },
+            'code':
+            code,
```

Comment:
The formatter wanted this on a new line?

Reply:
Must have. I bumped it up again; we'll see what happens.